clang/lib/Basic/Targets/BPF.h

```diff
@@ -112,6 +112,8 @@ class LLVM_LIBRARY_VISIBILITY BPFTargetInfo : public TargetInfo {
     StringRef CPUName(Name);
     return isValidCPUName(CPUName);
   }
+
+  bool hasExtIntType() const override { return true; }
```

Comment:
maybe this should depend on whether the `solana` feature is enabled? The base class `TargetInfo` method `hasExtIntType` returns `false`.

Reply:
Yes, it should. In fact I've tried to upstream this patch and got the feedback that these types need enabling in the linux kernel. I started working on this https://www.spinics.net/lists/netdev/msg711156.html but then I got the feedback "why do we need this in the kernel" which I didn't have a good answer.

Making this solana-only sidesteps that entire discussion.

Reply:
Fixed